sdk/python/lib/test/test_next_serialize.py

```diff
@@ -530,7 +550,7 @@ async def test_apply_does_not_propagate_secret_on_unknown_unknown_output_during_
         settings.SETTINGS.dry_run = True
 
         out = self.create_output(0, is_known=False)
-        r = out.apply(lambda v: self.create_output("inner", is_known=false, is_secret=True))
+        r = out.apply(lambda v: self.create_output("inner", is_known=False, is_secret=True))
```

Comment:
Noticed this when I had the file open in VS Code with Pylance installed.

Reply:
The lambda passed to `apply` is not called because `settings.SETTINGS.dry_run = True`.  If `settings.SETTINGS.dry_run` was set to `False`, then it fails as you'd expect with:

```
NameError: name 'false' is not defined
```